features/collection.feature

```diff
@@ -0,0 +1,9 @@
+Feature:
+  In order to use Postman with API Platform
+  as developper
+  then i should be able to generate a Postman collection file
+
+  Scenario: Generates a Postman collection with 1 folder and 5 requests
```

Comment:
Can you test it with multiple entry points? This will ensure there are 2 folders and requests are correctly set to related folder